tests/data/placebo/test_ecs_task_def/ecs.DescribeTaskDefinition_1.json

```diff
@@ -2,98 +2,70 @@
     "status_code": 200,
     "data": {
         "taskDefinition": {
-            "taskDefinitionArn": "arn:aws:ecs:us-east-1:644160558196:task-definition/ecs-read-only-root:1",
+            "taskDefinitionArn": "arn:aws:ecs:us-east-1:644160558196:task-definition/c7n:1",
```

Comment:
this file didn't need to be re-recorded, it just needed a list tags api call added to its output.